bundles/io.openliberty.tools.eclipse.ui/plugin.xml

```diff
@@ -59,12 +59,22 @@
             id="io.openliberty.tools.eclipse.project.startInContainer.command"
             name="Liberty Start in Container"
             categoryId="io.openliberty.tools.eclipse.command.category">
-      </command>      
+      </command>  
```

Comment:
One last thing. You also need to change at the very least the documentations ([user-guide.md](https://github.com/OpenLiberty/liberty-tools-eclipse/blob/main/docs/user-guide.md)), which includes text and images to reflect these new additions.
All other collateral would also need to be updated to reflect these new changes (blogs/videos/etc).

Reply:
I don't think we need to immediately redo all the collateral.  Eventually the next round will include the new options. 

For this PR?  Ideally the user guide would be updated but not sure it's 100% required.

Reply:
I will update the user-guide.md. The blogs and videos are ok leaving as is since they are more snapshots of the behavior at that current time, bound to change with future development.